### DISCUSSION TITLE
Question for @mxmou

### DISCUSSION THREAD

**Discussion by WorldLanguages** [ScratchAddons/ScratchAddons#675] [Help] [answered]
Just curious, can you give us an estimated date for the release of your dark theme/mode for the website addon?
cc @mxmou

**Comment by mxmou** [comment, marked as answer]
Probably in the next few days. I'm sorry that you have to wait so long, I could have finished it before but I have other things to do (school etc.).

**Reply by WorldLanguages** [reply]
@mxmou I got a profile comment about 2.0→3.0: https://scratch.mit.edu/users/World_Languages/#comments-145327401

**Reply by mxmou** [reply]
@WorldLanguages That's a bug. `addons/scratchr2/scratchwww.css` changes the color of `a:link` and `a:visited`, but not just `a` (links without hrefs, which the "See more replies" button is, aren't selected by the `:link` pseudo-class).

Note that this is only the case in scratch-www (project) comments. The same button in scratchr2 comments is styled correctly.
